kernel/os/src/arch/cortex_m33/os_fault.c

```diff
@@ -213,6 +217,10 @@ os_default_irq(struct trap_frame *tf)
     uint32_t orig_sp;
 #endif
 
+#if MYNEWT_VAL(MEMPROTECT_CLEAR_DATA)
```

Comment:
if MTB buffer is used, having this call before mtb_stop() is called can clobbers most of MTB content with loop data that is not relate to crash.

Reply:
I see, I will move it to after mtb_stop